README.md

```diff
@@ -267,3 +267,83 @@ npm install --save \
 ##### Example
 
 There are several parts to React Router, so it is recommended to read through the documentation available here: https://github.com/reactjs/react-router/tree/master/docs#table-of-contents
+
+#### CSS Bundling
```

Comment:
Add this to the table of contents under General and link it, `- [CSS Bundling](#css-bundling)`.